Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM python:3.7
+FROM python:3.7.11-slim
 
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y python-pip python3.7 python3-setuptools autoconf libtool pkg-config python3-dev build-essential
+RUN apt-get update && apt-get install
 
 COPY ./requirements.txt /app/requirements.txt
+COPY ./exchange.py /app/exchange.py
 
-COPY . /app
+#COPY . /app
 
 # ENV AUCTION_DEBUG_MODE False
-ENV FLASK_RUN_HOST=0.0.0.0
+# ENV FLASK_RUN_HOST=0.0.0.0
 
-EXPOSE 2222
+#EXPOSE 2222
 
-WORKDIR /app
+RUN pip install -r requirements.txt
```

Comment:
A mix of absolute and relative path usages confused me a bit. Usually, I will use `WORKDIR` when I plan to reference everything under it using relative path. It's very minor thing, you don't need to change.